index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>판다마켓</title>
+</head>
+
+<body>
+    <header>
+        <a class="goto_root" href="./index.html">
+            <img src="./icons/Property 1=Variant3.svg" alt="main logo">
+        </a>
+        <a class="button-login" href="./login.html">
+            로그인
+        </a>
+    </header>
+
+    <section class="banner-top">
+        <div class="banner-top-container">
+            <div class="banner-top-component">
+                <div class="text">
+                    일상의 모든 물건을<br>
+                    거래해 보세요
+                </div>
+                <a class="goto-items" href="./items.html">
+                    구경하러 가기
+                </a>
+            </div>
+            <img class="banner-top-image" src="./images/Img_home_top.svg" alt="banner image">
+        </div>
+    </section>
+
+    <main>
+        <section class="section1">
+            <img class="section-image" src="./images/Img_home_01.svg" alt="hot item image">
+            <div class="section1-component">
+                <div class="tag">Hot item</div>
+                <div class="text">인기 상품을<br>확인해 보세요</div>
+                <div class="text-info">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</div>
```

Comment:
```suggestion
                <div class="tag">Hot item</div>
                <div class="text">인기 상품을<br>확인해 보세요</div>
                <div class="text-info">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</div>
```

화면상 이 영역이 모두 영역을 나타내는 태그로 쓰여야 하는지,
아니면 어떤 역할을 하는지 생각해보시고 적절한 태그를 사용해보세요.